EasyPost.Net/Client.cs

```diff
@@ -34,59 +34,78 @@ public int RequestTimeoutMilliseconds
             set => _requestTimeoutMilliseconds = value;
         }
 
-        private string UserAgent => $"EasyPost/v2 CSharpClient/{_libraryVersion} .NET/{_dotNetVersion}";
+        private string UserAgent => $"EasyPost.Net/v2;CSharpClient/{_libraryVersion};.NET/{_dotNetVersion}";
```

Comment:
Can we leave the user agent as it was? All other libraries follow this pattern.

Reply:
Resolved in [d903ec7](https://github.com/EasyPost/easypost-csharp/pull/214/commits/d903ec7476be02f732e6eb4245a85595fc72df59)